### ISSUE TITLE
SPF

### ISSUE THREAD

**Issue by allella** [sendgrid/docs#2427]
I was able to setup [whitelabel domains](https://sendgrid.com/docs/Classroom/Deliver/Delivery_Introduction/all_you_need_to_know_about_whitelabeling.html) with automatic security. This was all done using the CNAME records provided in the web interface (Settings -> Whitelabel -> Domains) and everything is validated and green checkboxes.

Historically, I'm familiar with setting SPF in TXT records so I looked around to see if I ALSO needed to add include:sendgrid.net to my existing TXT. 

This [glossary page for SPF](https://sendgrid.com/docs/Glossary/spf.html) actually show an example of adding a root-level SPF record of include:sendgrid.net.  So, I'm a bit confused if include:sendgrid.net is still necessary, or if the whitelabel and automatic security make this traditional step unnecessary.

To further add to the confusion, there's a third document, [another page about whitelabeling](https://sendgrid.com/docs/Classroom/Deliver/Sender_Authentication/spf_records_explained.html), which suggests adding an SPF like include:u123456.wl.sendgrid.net to the TXT record.

In short, I'm wondering if the whitelabel domain CNAMEs is all you need to do to make SPF happy. Or, is there a situation where you also need to add one of the TXT records mentioned in the other two documents.

**Comment by thinkingserious** [comment]
Hello @allella,<br /><br/>Thanks for your suggestion or question!<br />                                              <br />In an attempt to clear out our backlog,                                              we are closing out issues over 90 days old.                                             If this is still an issue, please feel free to reopen it.                                             <br /><br />Team SendGrid DX-Docs

**Comment by sntran** [comment]
I'm also confused by the two articles on SPF record, and not sure which one should I use.
